### PR TITLE
fix: property name and beta flag spacing

### LINKED ISSUE #1008: API type: Spacing between property name and beta flag got lost - 1
image

There should be spacing between the type name and beta flag.  It must have accidentally disappeared as part of some refactoring or other change

### PR DESCRIPTION
**Pull request by davifantasia**
Closes #1008 

Replaces #1037

### CODE REVIEW

**Review by timonrey — APPROVED** [review]
👍🏼

File: packages/gatsby-theme-api-docs/src/components/type/properties/rows/name-type.js [hunk 1]
```diff
@@ -35,33 +36,37 @@ const NameType = (props) => {
...
+                  /
+                </span>
+              </Markdown.InlineCode>
+            ) : (
+              <Markdown.InlineCode>{props.property.name}</Markdown.InlineCode>
+            )}
+            {props.property.required && <Required />}
+          </SpacingsInline>
+          {props.property.beta && (
+            <BetaWrapper>
```
Comment (nkuehn) [inline packages/gatsby-theme-api-docs/src/components/type/properties/rows/name-type.js:65]: Why is the beta flag not part of the inline spacing wrapper but outside?

**Review by nkuehn — COMMENTED** [review]
@davifantasia to be honest I find the effect in the VRTs a bit suspicious.   The implementation seems to affect all layouts, even if there is no beta flag present at all.  

Are we really sure it was never correct before?

File: packages/gatsby-theme-api-docs/src/components/type/properties/rows/name-type.js (hunk 1 above)
Comment (davifantasia) [reply packages/gatsby-theme-api-docs/src/components/type/properties/rows/name-type.js:63]: Thanks for catching that, I used an unnecessary extra `SpacingsInline`

**Pull request merged**